### ISSUE TITLE
Fix to avoid return empty result if user dict has empty line

### ISSUE THREAD

**Issue by clsung** [monpa-team/monpa#10]
```--- __init__.py~	2020-04-14 22:09:35.000000000 +0800

+++ __init__.py	2020-04-14 22:09:55.000000000 +0800
@@ -246,7 +246,7 @@
     # empty previous userdict
     _userdict = []
     for input_item in io.open(pathtofile, 'r', encoding="utf-8").read().split("\n"):
-        _userdict.append(input_item.split(" "))
+        if input_item: _userdict.append(input_item.split(" "))

 def findall(p, s):
     ''' Yields all the positions of the pattern p in the string s```

**Comment by windyeh** [comment]
thanks, scheduled to revamp by next release.

**Comment by windyeh** [comment]
fix at v0.3.2 release, thanks.
